transformations/americanize_britishize_english/transformation.py

```diff
@@ -0,0 +1,131 @@
+import requests
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def britishize_americanize(string, final_dict):
+    """
+    Parameters:
+        string(str): original string
+        final_dict(dict): dictionary with all the different possible words in american and british english
+    Returns:
+        str: String after replacing the words
+    """
+    string = " ".join([final_dict.get(word, word) for word in string.split()])
```

Comment:
How would it handle plural forms. Let's say will it transform queues to lines and vice versa?

Reply:
Agree ^
In my MR, I used Inflect Library : https://pypi.org/project/inflect/ to pluralize + singularize words.

Reply:
the dictionaries have individual entries for singular and plural words. For example:-
"airplane":"aeroplane",
"airplanes":"aeroplanes",

Reply:
Ok I guess I see the issue. In british_american_dict there are indeed entiries for plural and singular forms but in "difference_british_to_american" this is not the case.